.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
VSCode has an extension called Prettier which helps auto-format your code on save. This file just adds some configuration for that.

Reply:
No problem, was just making you aware of it. I added this config so that my local settings wouldn't change the quotes and add semi-colons everywhere. It'd be make it a lot harder to show you exactly what I changed here.